### PR TITLE
F #30 receipt search

### PR DESCRIPTION
**Pull request by querqueq**
Implemented the receipt search user story.

### CODE REVIEW

File: src/main/resources/templates/staff/customerCreate.html [hunk 1]
```diff
@@ -1,82 +1,179 @@
...
+					<label for="billingAddress.state"
+						class="col-2 col-form-label required">Land</label>
+					<div class="col-10">
+						<input id="billingAddress.state" class="form-control" type="text"
+							th:field="*{billingAddress.state}" required>
+					</div>
+				</div>
+			</fieldset>
+			<div class="form-group row">
+				<label><span style="color: red;">*</span> indicates required
```
Comment (a-kraschitzer) [inline src/main/resources/templates/staff/customerCreate.html]: in Deutsch uebersetzen

File: src/main/resources/templates/staff/customerCreate.html [hunk 2]
```diff
@@ -1,82 +1,179 @@
...
+					fields</label>
+			</div>
+			<button type="submit" class="btn btn-primary">Submit</button>
```
Comment (a-kraschitzer) [inline src/main/resources/templates/staff/customerCreate.html]: Uebersetzun Deutsch

File: src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/AddressEntity.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.domain.data;
+
```
Comment (a-kraschitzer) [inline src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/AddressEntity.java]: Add java docs

File: src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/CustomerEntity.java [hunk 4]
```diff
@@ -3,30 +3,31 @@
 import java.math.BigDecimal;
```
Comment (a-kraschitzer) [inline src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/CustomerEntity.java]: java docs

File: src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/ReceiptEntity.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.domain.data;
+
```
Comment (a-kraschitzer) [inline src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/ReceiptEntity.java]: JavaDocs

File: src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/AddressEntity.java (hunk 3 above)
Comment (querqueq) [reply src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/domain/data/AddressEntity.java]: done

File: src/main/resources/templates/staff/customerCreate.html (hunk 2 above)
Comment (querqueq) [reply src/main/resources/templates/staff/customerCreate.html]: done

File: src/main/resources/templates/staff/customerCreate.html (hunk 1 above)
Comment (querqueq) [reply src/main/resources/templates/staff/customerCreate.html]: done

**Pull request merged**